index.bs

```diff
@@ -28,9 +28,16 @@ urlPrefix: https://w3c.github.io/sensors; spec: GENERIC-SENSOR
     text: sensor
     text: latest reading
     text: default sensor
-    text: construct a sensor object; url: construct-sensor-object
     text: sensor type
     text: mitigation strategies; url: mitigation-strategies
+    text: local coordinate system
+urlPrefix: https://w3c.github.io/accelerometer; spec: ACCELEROMETER
```

Comment:
Let's put a note in Accelerometer, why we are using SpatialSensor in Accelerometer spec.